### ISSUE TITLE
[BUG] failure eslint error with DaisyUI

### ISSUE THREAD

**Issue by nonameolsson** [francoismassart/eslint-plugin-tailwindcss#150]
**Describe the bug**
When linting with Trunk.io (using Eslint linter) an error is thrown. I am not sure if this is related with this plugin, or if it is related with DaisyUI.

**To Reproduce**
1. Add eslint-plugin-tailwindcss
2. Add trunk
3. Add DaisyUI

**Expected behavior**
No lint error should occur. 

**Screenshots**
![image](https://user-images.githubusercontent.com/1493018/177493281-89ef9c02-651a-45cd-b772-02bee13f2044.png)

**Environment (please complete the following information):**
 - OS: [macOS]
 - Softwares + version used:
     -  "daisyui": "^2.18.0",
     - "eslint-plugin-tailwindcss": "^3.6.0",
     - Trunk: 0.14.1-beta
     - eslint@8.18.0

**Additional context**

**eslint config file or live demo**
https://user-images.githubusercontent.com/1493018/177493802-1d98d00c-674c-486c-a133-ce35a25de57d.mp4

Trunk.yml
```yml
version: 0.1
cli:
  version: 0.14.1-beta
lint:
  ignore:
    - linters: ["eslint"]
      paths:
        # Ignore generated files
        - build/**
        # Trunk
        - .trunk/**
        - .trunk/out/*.yaml
  enabled:
    - actionlint@1.6.15
    - eslint@8.18.0
    - gitleaks@8.8.11
    - hadolint@2.10.0
    - markdownlint@0.31.1
    - prettier@2.7.1
    - shellcheck@0.8.0
    - shfmt@3.5.0
    - svgo@2.8.0
    - taplo@release-taplo-cli-0.6.8
```

.eslintrc.js
```js
/**
 * @type {import('@types/eslint').Linter.BaseConfig}
 */
module.exports = {
  extends: [
    '@remix-run/eslint-config',
    '@remix-run/eslint-config/node',
    '@remix-run/eslint-config/jest-testing-library',
    'plugin:tailwindcss/recommended',
    'prettier'
  ],
  plugins: ['tailwindcss'],
  // we're using vitest which has a very similar API to jest
  // (so the linting plugins work nicely), but it means we have to explicitly
  // set the jest version.
  settings: {
    jest: {
      version: 27
    }
  },
  rules: {
    'tailwindcss/no-custom-classname': 'off'
  }
}
```

tailwind.config.js
```js
module.exports = {
  content: ['./app/**/*.{ts,tsx,jsx,js}'],
  theme: {
    extend: {}
  },
  plugins: [
    require('@tailwindcss/typography'),
    require('@tailwindcss/forms'),
    require('@tailwindcss/line-clamp'),
    require('daisyui')
  ]
  // daisyui: {
  //   darkTheme: 'dark'
  // }
}
```

**Comment by dapirian** [comment]
A little more info:

The output of eslint with this plugin when using `--format json` is:

```
      [35m🌼 daisyUI components 2.17.0[0m [0m https://github.com/saadeghi/daisyui
        [32m✔︎ Including:[0m [0m base, components, themes[29], utilities
        

      [35m🌼 daisyUI components 2.17.0[0m [0m https://github.com/saadeghi/daisyui
        [32m✔︎ Including:[0m [0m base, components, themes[29], utilities
        
      [{"filePath":"/workspace/linjen/app/components/navbar.tsx","messages":[],"suppressedMessages":[],"errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":[]},{"filePath":"/workspace/linjen/app/components/page.tsx","messages":[],"suppressedMessages":[],"errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":[]},{"filePath":"/workspace/linjen/app/routes/timeline/$timelineId.tsx","messages":[],"suppressedMessages":[],"errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":[]}]
```

Note that the `daisyUI` lines are _not_ json. It shouldn't print not-json when using `--format json`

**Comment by francoismassart** [comment]
Hi @nonameolsson 
sorry for the delay 😅 
I don't know much about trunk, I recently published a new beta version which should support Tailwind CSS up to 3.2.3 as well as fixed ordering for older Tailwind CSS version.

If by chance the bug you are experiencing is caused by a side effect, there is a chance it could be fixed as well.

Could you try it out via:

1. `npm i -D eslint-plugin-tailwindcss@3.7.0-beta.0`
2. Test it on your own projects
3. Share your feedback in the comments ?

Thank you

**Comment by francoismassart** [comment]
Version 3.7.0 has been released and it should fix the issue

`npm i -D eslint-plugin-tailwindcss`
or
`npm i -D eslint-plugin-tailwindcss@3.7.0`

If the issue persists, please open a new ticket.

Cheers.
